city_scrapers/spiders/wayne_state_university.py

```diff
@@ -0,0 +1,95 @@
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+import re
+
+
+class WayneStateUniversitySpider(CityScrapersSpider):
+    name = "wayne_state_university"
+    agency = "Wayne State University"
+    timezone = "America/Chicago"
+    allowed_domains = ["bog.wayne.edu"]
+    start_urls = ["http://bog.wayne.edu/"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        
+        #one large all encompassing meeting
+        meeting_dates = response.xpath('//*[@id="content"]/div[1]/div/h2').re('((?:January|Feburary|March|April|May|June|July|August|October|September|November|December)\s+\d{1,2},\s+\d{4})')
+        for item in meeting_dates:
```

Comment:
When there's a link in the `li` element, you'll want to `yield` a request to another page that parses detail pages (one example here: https://bog.wayne.edu/meetings/2019/09-20)

You can check if the `item` has a link in it by seeing if `len(item.css("a"))` is more than 0